doc/getting-started.md

```diff
@@ -92,46 +93,51 @@ curl \
 "$2b$12$CpJZx5ooxM11bCFXT76/z.o6HWs2sPJy4iP8.xCZGmM8jWXUXJZ4K"
 ```
 
-Then create a new user entry in the database.  You will need to provide the
-password hash from the previous command with special characters escaped.  In
-particular, `$` characters need to be escaped with `\$`.  For example, a Mongo
-shell command to create a user entry with the name `bob` and the hash from the
-example above would be:
-
-```
-db.user.insert({username: 'bob', hashed_password: '\$2b\$12\$VtfVij6zz20F/Qr0Ri18O.11.0LJMMXyJxAJAHQbKU0jC96eo2fr.', active: true})
-```
-
-This can be run directly from a terminal via `docker-compose exec`:
+Then create the admin user entry in the database manually via `docker-compose
+exec` to access the database container.  You will need to provide the password
+hash from the previous command with special characters escaped.  In particular,
+`$` characters need to be escaped with `\$`.  For example:
 
 ```
 $ docker-compose exec db /bin/mongo kernelci --eval \
-  "db.user.insert({username: 'bob', hashed_password: '\$2b\$12\$VtfVij6zz20F/Qr0Ri18O.11.0LJMMXyJxAJAHQbKU0jC96eo2fr.', active: true})"
+  "db.user.insert({\
+    username: 'admin', \
+    hashed_password: '\$2b\$12\$VtfVij6zz20F/Qr0Ri18O.11.0LJMMXyJxAJAHQbKU0jC96eo2fr.', \
+    active: true, \
+    is_admin: 1\
+})"
 MongoDB shell version v5.0.3
 connecting to: mongodb://127.0.0.1:27017/kernelci?compressors=disabled&gssapiServiceName=mongodb
 Implicit session: session { "id" : UUID("84a62d1b-4b06-4631-8227-413964826100") }
 MongoDB server version: 5.0.3
 WriteResult({ "nInserted" : 1 })
 ```
 
-Even though after creating the above user, we would be able to run API, ideally, an admin user needs to be created first (using terminal), and then additional users can be created with the `/user` endpoint or kernelci-core command line tools (`kci_data`). The instructions are described [here](https://kernelci.org/docs/api/api-details/#users).
+> **Note** For more details about how to create users via the raw API, see the
+> [API documentation](/docs/api/api-details/#users)
 
```

Comment:
If we drop the `Create an admin user` section from the `api-details`, we need to add the below line at the end:

```Regular users can also be created this way without the `is_admin` attribute.```

Reply:
OK, but I don't think we should drop the section from the API documentation.

Reply:
If we don't want to drop the whole section from `api-details`, we can remove the description of creating an admin user from it and add a link to the `getting-started` page. In case we want to add some details in the future, we can use the same section.

Reply:
OK, but this is about the detailed API documentation. This PR is to update the "getting started" section which needs to have instructions about how to create an initial admin user. So if the changes in this PR look fine we can have them merged and then rework the detailed API documentation as a follow-up.

Reply:
Yes, makes sense. Let me merge this.